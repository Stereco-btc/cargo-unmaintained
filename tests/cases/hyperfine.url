https://github.com/sharkdp/hyperfine
