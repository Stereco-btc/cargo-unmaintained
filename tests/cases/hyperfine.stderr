Scanning 159 packages and their dependencies (pass --verbose for more information)
